src/org/traccar/database/NotificationManager.java

```diff
@@ -54,6 +57,15 @@ public void updateEvent(Event event, Position position) {
         }
 
         Set<Long> users = Context.getPermissionsManager().getDeviceUsers(event.getDeviceId());
+        Device device = Context.getIdentityManager().getDeviceById(event.getDeviceId());
+        if (device != null && device.getAttributes().containsKey("notifyOnly")) {
```

Comment:
I just read original thread and it doesn't make sense to me. What's the use case for this? I'll ask on the original issue.